app/assets/javascripts/site.js.coffee

```diff
@@ -56,6 +56,6 @@ $(document).ready ->
     $(".site_header").addClass("site_header_background");
   Site.setClipHeight($(this));
 
-  if (window.location.pathname && window.location.pathname.includes('login'))
-    $(".site-login").addClass("site-login-hidden")
+  # if (window.location.pathname && window.location.pathname.includes('login'))
```

Comment:
Hmm my preference for UI/UX will be consistency, where I expect the public-facing page's navigation bar to show the same content across all webpages.
However, this may not necessarily be the best approach! I am open to adopting any standardized UI/UX guidelines for this project.